packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,31 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Use custom ScrollController when scrollController is set', (WidgetTester tester) async {
+        final ScrollController customController = ScrollController(initialScrollOffset: 10);
+        await tester.pumpWidget(
+          MaterialApp(
+            home: ReorderableListView(
+              scrollController: customController,
+              children: const <Widget>[
+                SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
+                SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
+              ],
+              onReorder: (int oldIndex, int newIndex) { },
+            ),
+          )
+        );
```

Comment:
Wrap the `ReorderableListView` by a smaller box so that you can test scrolling the list view and seeing a measurable change. For example, if you constrain the list view to a height of 200, if you have three boxes in the list view at heights of 100.0 each, you can test scrolling the list view since the list view can be hidden:
```dart
await tester.pumpWidget(
  MaterialApp(
    home: Scaffold(
      body: SizedBox( // Constrains the height of the ListView to 200.0
        height: 200,
        child: ReorderableListView(
          scrollController: customController,
          children: const <Widget>[
            SizedBox(height: 100.0, child: Text('A'), key: firstBox),
            SizedBox(height: 100.0, child: Text('B'), key: secondBox),
            SizedBox(height: 100.0, child: Text('C'), key: thirdBox),
          ],
          onReorder: (int oldIndex, int newIndex) { },
        ),
      ),
    ),
  ),
);
```